libs/admin-console/feature/src/lib/admin-console.component.ts

```diff
@@ -1,25 +1,31 @@
 import { Component, OnInit } from '@angular/core';
 import { API } from "../../../../shared/api/api.service";
+import { Store } from "@ngxs/store";
+import { Observable } from 'rxjs';
+import { JsonPipe } from '@angular/common';
 
 @Component({
   selector: 'the-au-pair-admin-console',
   templateUrl: './admin-console.component.html',
   styleUrls: ['./admin-console.component.scss'],
 })
 export class AdminConsoleComponent implements OnInit{
-
   auPairs : any[] = [];
+  idNum : any;
+  
+  constructor(private serv: API, public store:Store) {
+    this.idNum = this.store.snapshot().user.id;
+    console.log(this.store.snapshot().user.id);
```

Comment:
I think we can remove these console.logs otherwise out cleanup at the end will be monumental